docs/api/ShallowWrapper/invoke.md

```diff
@@ -0,0 +1,41 @@
+# `.invoke(invokeName[, ...args]) => Any`
+
+Invokes a function prop.
+
+#### Arguments
+
+1. `propName` (`String`): The function prop to be invoked
+2. `...args` (`Any` [optional]): Arguments that will be passed to the prop function
```

Comment:
Using a spread here prevents ever being able to pass options.

The pattern we went with in `renderProp` was to return a new function, which could then be invoked with a variable number of arguments. What do you think about doing that here?